### ISSUE TITLE
lr_scheduler has to be ReduceLROnPlateau?

### ISSUE THREAD

**Issue by puppywst** [tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation#49]
I used stepLR instead of ReduceLROnPlateau, but only got mAP 43% with scale 1 on 1160 images of coco val2014. However, your released model achieves 52.3% .
I observed the last heatmap loss, mine stays at 24, but yours at 19. Does lr_scheduler really infulence so much?

**Comment by tensorboy** [comment]
Learning rate is the most important hyper-parameter of deep learning optimization, if you want to use multi-step you need find which step to decay carefully. :)
